### PR TITLE
Fw 5329 5500 word phrase editing redirect

### PR DESCRIPTION
**Pull request by Cara-Barter**
### Description of Changes

1. edit word/phrase opens in a new tab.
2. upon save word or phrase, redirects to public side

### Checklist

- [ ] README / documentation has been updated if needed
- [x] PR title / commit messages contain Jira ticket number if applicable
- [ ] Tests have been updated / created if applicable

### Reviewers Should Do The Following:

- [x] Review the changes here
- [ ] Pull the branch and test locally

### Additional Notes

N/A

### CODE REVIEW

File: src/components/DictionaryCrud/DictionaryCrudData.js [hunk 1]
```diff
@@ -23,8 +24,10 @@ function DictionaryCrudData() {
     id: entryId,
     edit: true,
   })
+  const type = data?.type === TYPE_PHRASE ? PHRASES : WORDS
+
   const { onSubmit: createEntry } = useDictionaryEntryCreate()
-  const { onSubmit: updateEntry } = useDictionaryEntryUpdate()
+  const { onSubmit: updateEntry } = useDictionaryEntryUpdate(entryId, type)
```
Comment (gmcauliffe) [inline src/components/DictionaryCrud/DictionaryCrudData.js:30]: As these props are not being read or used by `useDictionaryUpdate`, best to remove them before merging.

**Review by gmcauliffe — CHANGES_REQUESTED** [review]
One small change and a suggestion when you have a moment. Looking good!

File: src/components/DictionaryCrud/DictionaryCrudData.js (hunk 1 above)
Comment (Cara-Barter) [reply src/components/DictionaryCrud/DictionaryCrudData.js:30]: oops, missed that one

File: src/common/dataHooks/useDictionaryEntry.js [hunk 2]
```diff
@@ -44,16 +55,10 @@ export function useDictionaryEntryCreate() {
     })
   }
 
-  const { user } = useUserStore()
```
Comment (gmcauliffe) [inline src/common/dataHooks/useDictionaryEntry.js:47]: Good call on removing the Assistant conditional logic with this update 👍

**Review by gmcauliffe — APPROVED** [review]
LGTM 👍

**Pull request merged**